eprihlaska/views.py

```diff
@@ -320,11 +320,11 @@ def admissions_waivers():
         form[grade_key].grade_third_year.label.text = consts.GRADE_THIRD_YEAR[los] # noqa
 
     further_study_info_constraints = {
-        'matura_mat_grade': ['AIN',
-                             'upFYIN', 'upINAN', 'upINBI', 'upINGE', 'upINCH', 'upMADG' 'upMAFY',
```

Comment:
@tvinar Note the missing comma at the end of this line...

Reply:
Who would have noticed :)